fusor-ember-cli/app/components/text-f.js

```diff
@@ -56,25 +65,24 @@ export default Ember.Component.extend({
     'invalidIsAlphaNumeric',
     'invalidIsHostname',
     'invalidNetworkRange',
+    'invalidCIDRNotation',
     function() {
       return (this.get('showValidationError') &&
-              (Ember.isPresent(this.get('errors.name')) ||
+               (Ember.isPresent(this.get('errors.name')) ||
                this.get('doesntMatchPassword') ||
                this.get('passwordTooShort') ||
                this.get('validIsRequiredAndBlank') ||
                this.get('validIsUnique') ||
                this.get('invalidIsAlphaNumeric') ||
                this.get('invalidIsHostname') ||
-               this.get('invalidNetworkRange')
-              )
+               this.get('invalidNetworkRange') ||
+               this.get('invalidCIDRNotation'))
```

Comment:
@isratrade @eriknelson This is not to address now and the PR looks good to me, but it does seem like this textfield is growing rather large and has lots of options for fields that aren't usually relevant.  What do you guys think about (in the future) refactoring the text-f to take in a list of validation objects [{isValidFn: functoin, displayError: string }, ..] into the component.  We could put the common validation functions and error message constants in the validation utils sot hey don't pollute our controllers.  Then we could just use {{text-f validations=passwordValidations}} etc.